package.json

```diff
@@ -38,7 +38,8 @@
     "raf": "^3.4.0",
     "stylelint": "^9.3.0",
     "stylelint-config-standard": "^18.0.0",
-    "surge": "^0.20.3"
+    "surge": "^0.20.3",
+    "redux-mock-store": "^1.2.2"
```

Comment:
I think it would be better to use `theforeman/test` package or an alternative than introducing another npm package that doesn't exist in packaging 2.3

Reply:
@theforeman/packaging This package is also used in foreman 2.3. Is it a problem that it doesn't exist in packaging 2.3 if it's only used in testing?

Reply:
If it's for tests only, then we don't need it packaged, just added to the exclude list (if it's not there yet)

Reply:
https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L37
Looks like it's there

Reply:
Then you're safe from the packaging 👻!

Reply:
Awesome!